pkg/authorization/cache/strategy.go

```diff
@@ -0,0 +1,35 @@
+package cache
```

Comment:
rename this file.  strategy.go is typically reserved for things that implement a RESTStrategy.  put this in a util.go or something equivalent in name.